# goal definition
string arm_safe_position
string source_location
string destination_location
---
# result definition
bool success
---
# feedback
string current_state
string text
